<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAirportCompleter</class>
 <widget class="QFrame" name="CAirportCompleter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Airport completer</string>
  </property>
  <layout class="QHBoxLayout" name="hl_AirportCompleter" stretch="1,3,3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="le_Icao">
     <property name="placeholderText">
      <string>ICAO</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_Name">
     <property name="placeholderText">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_Location">
     <property name="placeholderText">
      <string>location</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_Icao</tabstop>
  <tabstop>le_Name</tabstop>
  <tabstop>le_Location</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
